libs/blocks/text/text.css

```diff
@@ -109,10 +109,11 @@
 }
 
 /* Legal */
-.legal.text-block .foreground {
+.legal.text-block {
```

Comment:
Hi @drashti1712,

why does the .foreground need to be removed here?
Saw the padding was working properly.

Reply:
Hi @elaineskpt, I removed it as we are also considering an authoring change as mentioned in the comments for this issue: https://jira.corp.adobe.com/browse/MWPW-132398
So if we add another row to accommodate this use-case then, we would need padding on the outer div and not on each row (foreground). 
And even if we do not accommodate this change, adding a padding to the outer div instead of on the row seemed logical.